### PR TITLE
Add platforms annotation to several task

### PR DESCRIPTION
**Pull request by barthy1**
# Changes
Annotation about `linux/amd64`,`linux/s390x`,`linux/ppc64le` platforms was added to the latest versions of the  `tkn`, `git-cli`, `pull-request` and `kubeconfig-creator` tasks.

Tested for all mentioned platforms using e2e test provided with the tasks.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Follows the [authoring recommendations](https://github.com/tektoncd/catalog/blob/main/recommendations.md)
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality of task changed or new task added)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [ ] Complies with [Catalog Organization TEP][TEP], see [example]. **Note** [An issue has been filed to automate this validation][validation]
  - [ ] File path follows  `<kind>/<name>/<version>/name.yaml`
  - [ ] Has `README.md` at `<kind>/<name>/<version>/README.md`
  - [ ] Has mandatory `metadata.labels` - `app.kubernetes.io/version` the same as the `<version>` of the resource
  - [ ] Has mandatory `metadata.annotations` `tekton.dev/pipelines.minVersion`
  - [ ] mandatory `spec.description` follows the convention

          ```

          spec:
            description: >-
              one line summary of the resource

              Paragraph(s) to describe the resource.
          ```

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

---

[TEP]: https://github.com/tektoncd/community/blob/master/teps/0003-tekton-catalog-organization.md
[example]: https://github.com/tektoncd/catalog/tree/master/task/git-clone/0.1
[validation]:  https://github.com/tektoncd/catalog/issues/413

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]

File: task/tkn/0.2/tkn.yaml [hunk 1]
```diff
@@ -9,6 +9,7 @@ metadata:
     tekton.dev/pipelines.minVersion: "0.17.0"
     tekton.dev/tags: cli
     tekton.dev/displayName: "Tekton CLI"
+    tekton.dev/platforms: "linux/amd64,linux/s390x,linux/ppc64le"
```
Comment (vinamra28) [inline task/tkn/0.2/tkn.yaml:12]: should we add the same label for 0.1 as well?

File: task/git-cli/0.2/git-cli.yaml [hunk 2]
```diff
@@ -9,6 +9,7 @@ metadata:
     tekton.dev/pipelines.minVersion: "0.21.0"
     tekton.dev/tags: git
     tekton.dev/displayName: "git cli"
+    tekton.dev/platforms: "linux/amd64,linux/s390x,linux/ppc64le"
```
Comment (vinamra28) [inline task/git-cli/0.2/git-cli.yaml:12]: same should we add the same for git-cli 0.1?

File: task/tkn/0.2/tkn.yaml (hunk 1 above)
Comment (barthy1) [reply task/tkn/0.2/tkn.yaml:12]: done for git-cli and tkn tasks

**Pull request merged**